README.md

```diff
@@ -2,6 +2,10 @@
 
 This is the engine for [Webspaces](https://webspaces.space), which allows you to create 3D worlds using just HTML.
 
+## Notice
+
+If you want to get started with **webspaces.space** instead of the engine it works on, please go to [webpsace-sdk](https://github.com/webspace-sdk/webspace-sdk.github.io)
```

Comment:
this doesn't look right - the link is pointing to the repo for webspaces.space - I was suggesting we do something like:

`If you aren't interested working on the Webspace Engine, but wish to create your own webspace, please go to [webspaces.space]`

And dropping the other changes.

Reply:
Ho, okay, just a minute